### ISSUE TITLE
Enhancement: Add Euler number metric to structural QA report

### ISSUE THREAD

**Issue by jvarada** [poldracklab/mriqc#667]
https://www.biorxiv.org/content/early/2017/10/01/125161

**Comment by chrisfilo** [comment]
To calculate this metric one would have to run full `recon-all` from FreeSurfer. That would add 8-12h per scan, which might be just too much for most users.

**Comment by jvarada** [comment]
@chrisfilo Perhaps run it with an optional tag, rather than by default, like the ICA workflow?

**Comment by chrisfilo** [comment]
Yes that could work. Or providing already calculated FreeSurfer directory.

Perhaps it might be worthwhile exposing this variable in FMRIPREP reports.

**Comment by oesteban** [comment]
Moving this to fMRIPrep, where it could see an implementation one day.
